src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -3644,6 +3624,9 @@ void Creature::AtEngage(Unit* target)
         ai->JustEngagedWith(target);
     if (CreatureGroup* formation = GetFormation())
         formation->MemberEngagingTarget(this, target);
+
+    // Creatures with CREATURE_STATIC_FLAG_2_FORCE_PARTY_MEMBERS_INTO_COMBAT periodically force party members into combat
+    ForcePartyMembersIntoCombat();
```

Comment:
hmmmmmmm,. group updates packets also used to be updated here ( i meen health, auras of group members etc ) or it should be called from ::Heartbeat( ) ?